src/modules/layout/layout.component.html

```diff
@@ -41,4 +41,16 @@
   </stache-layout-sidebar>
 </template>
 
+<template #actionButtonLayout>
+  <stache-layout-action-button
+    pageTitle="{{pageTitle}}"
+    [showBackToTop]="showBackToTop"
+    [showBreadcrumbs]="showBreadcrumbs"
+    [breadcrumbsRoutes]="breadcrumbsRoutes"
+    [actionButtonRoutes]="actionButtonRoutes">
```

Comment:
Seeing this here, it's obvious that we need to rethink how we're passing navigation routes to each layout component. As I see it, there are only two types of layout routes: routes used for inter-page navigation (URLs) and routes used for intra-page navigation (page anchors). 

Inter-page navigation could include sidebar, action buttons, panels, whatever.

Intra-page navigation would be any page anchors for the current page.

We can leave it as `actionButtonRoutes` for this PR, but let's think about what would be the best name for these two categories for a new PR. (Perhaps we need the assistance of a tech writer: @blackbaud-brandonhare.)